pkg/volume/local/local.go

```diff
@@ -208,6 +207,26 @@ func (m *localVolumeMounter) SetUpAt(dir string, fsGroup *int64) error {
 		return nil
 	}
 
+	oldMount, err := mount.IsAlreadyMounted(dir)
+	if err != nil {
+		glog.Errorf("failed to check if disk %s has already been mounted(%v)", dir, err)
+		return err
+	}
+
+	if len(oldMount) > 0 {
```

Comment:
So this is handling the case where two pods simultaneously access the same volume.  But what about two pods one after another?  Do we want to protect in that case too?